source/features/quick-mention.tsx

```diff
@@ -6,10 +6,15 @@ import {ReplyIcon} from '@primer/octicons-react';
 import * as pageDetect from 'github-url-detection';
 import * as textFieldEdit from 'text-field-edit';
 
+import {wrap} from '../helpers/dom-utils';
 import features from '.';
 import {getUsername} from '../github-helpers';
 import onNewComments from '../github-events/on-new-comments';
 
+function prefixUserMention(userMention: string): string {
+	return userMention.startsWith('@') ? userMention : '@' + userMention;
+}
```

Comment:
Needed because the `alt` attribute of avatars next to review events doesn't include the `@` (i.e. it's just `fregante` instead of `@fregante`).

Reply:
```suggestion
function prefixUserMention(userMention: string): string {
	return '@' + userMention.replace('@', ''); // The alt may or may not have it #4859
}
```